src/radio/index.tsx

```diff
@@ -46,125 +45,114 @@ export interface RadioProperties extends ThemedProperties, FocusProperties {
+export const Radio = factory(function Radio({ properties, middleware: { focus, theme } }) {
+	const _uuid = uuid();
+	const {
+		aria = {},
+		checked = false,
+		classes,
+		disabled,
+		label,
+		labelAfter = true,
+		labelHidden,
+		name,
+		onBlur,
+		onFocus,
+		onOut,
+		onOver,
+		onValue,
+		readOnly,
+		required,
+		theme: themeProp,
+		valid,
+		value,
+		widgetId = _uuid
+	} = properties();
 
-		return [
-			css.root,
-			checked ? css.checked : null,
-			disabled ? css.disabled : null,
-			focus.containsFocus ? css.focused : null,
-			valid === true ? css.valid : null,
-			valid === false ? css.invalid : null,
-			readOnly ? css.readonly : null,
-			required ? css.required : null
-		];
-	}
+	const themeCss = theme.classes(css);
 
-	render(): DNode {
-		const {
-			aria = {},
-			checked = false,
-			disabled,
-			widgetId = this._uuid,
-			valid,
-			label,
-			labelAfter = true,
-			labelHidden,
-			theme,
-			classes,
-			name,
-			readOnly,
-			required,
-			value,
-			onOut,
-			onOver,
-			onBlur,
-			onFocus
-		} = this.properties;
-		const focus = this.meta(Focus).get('root');
-
-		const children = [
-			v('div', { classes: this.theme(css.inputWrapper) }, [
-				v('input', {
-					id: widgetId,
-					...formatAriaProperties(aria),
-					classes: this.theme(css.input),
-					checked,
-					disabled,
-					focus: this.shouldFocus,
-					'aria-invalid': valid === false ? 'true' : null,
-					name,
-					readOnly,
-					'aria-readonly': readOnly === true ? 'true' : null,
-					required,
-					type: 'radio',
-					value,
-					onblur: () => {
-						onBlur && onBlur();
-					},
-					onchange: this._onChange,
-					onfocus: () => {
-						onFocus && onFocus();
-					},
-					onpointerenter: () => {
-						onOver && onOver();
-					},
-					onpointerleave: () => {
-						onOut && onOut();
-					}
-				}),
-				v(
-					'div',
-					{
-						classes: this.theme(css.radioBackground)
-					},
-					[
-						v('div', { classes: this.theme(css.radioOuter) }),
-						v('div', { classes: this.theme(css.radioInner) })
-					]
-				)
-			]),
-			label
-				? w(
-						Label,
-						{
-							theme,
-							classes,
-							disabled,
-							focused: focus.containsFocus,
-							valid,
-							readOnly,
-							required,
-							hidden: labelHidden,
-							forId: widgetId,
-							secondary: true
-						},
-						[label]
-				  )
-				: null
-		];
-
-		return v(
-			'div',
-			{
-				key: 'root',
-				classes: this.theme(this.getRootClasses())
-			},
-			labelAfter ? children : children.reverse()
-		);
-	}
-}
+	return (
+		<div
+			key="root"
+			classes={[
+				themeCss.root,
+				checked ? themeCss.checked : null,
+				disabled ? themeCss.disabled : null,
+				focus.isFocused('root') ? themeCss.focused : null,
+				valid === false ? themeCss.invalid : null,
+				valid === true ? themeCss.valid : null,
+				readOnly ? themeCss.readonly : null,
+				required ? themeCss.required : null
+			]}
+		>
+			{!labelAfter && label ? (
```

Comment:
we've dropped the label before/after toggling from other widgets so could you just place the label after the checkbox please and remove the property.